source/reference/mongosync.txt

```diff
@@ -176,6 +176,38 @@ Global Options
 
    Prints ``mongosync`` version information to stdout.
 
+.. _c2c-cli-oidc:
+
+Microsoft Azure Managed Identities Environment Variables
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. include:: /includes/mongosync-and-oidc.rst
+
+To use Microsoft Azure Managed Identities for federated authentication,
+define these environment variables:
+
+.. option:: AZURE_TENANT_ID <id>
+
+   Sets the Azure tenant identifier. Example:
+   ``AZURE_TENANT_ID=08206ab8-16a0-406d-85e4-2f15f5620fac``
+
+.. option:: AZURE_APP_CLIENT_ID <id>
+
+   Sets the Azure application client identifier. Example:
+   ``AZURE_APP_CLIENT_ID=b6c835da-e536-425b-9405-64bc471e245b``
+
+.. option:: AZURE_CLIENT_ID <id>
+
+   Sets the Azure client identifier. Example:
+   ``AZURE_CLIENT_ID=f176d4eb-7dcd-4f66-bccf-aaa316ee61fd``
+
+.. option:: AZURE_FEDERATED_TOKEN_FILE <file>
+
+   Sets the Azure federated token file and path. Example:
```

Comment:
Likewise: were I a user, I would expect some guidance on where to find the value to plug in here.

(Also, “file and path” sounds to me like two different things; this looks rather like just a path.)